gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
```

Comment:
Please leave the gradle files and stuff on gitignore or not commit them.

Reply:
Yes, I added .idea and .graddle to gitignore, so this problem should not occur in the future.